src/pages/Browse/index.tsx

```diff
@@ -119,6 +130,16 @@ export class BrowseBase extends React.Component<Props> {
         }),
       );
     }
+
+    if (file && nextFilePath && !nextFileIsLoading && nextFile === undefined) {
+      dispatch(
+        _fetchVersionFile({
+          addonId: parseInt(match.params.addonId, 10),
```

Comment:
This should move to a variable so you can share it with the if statement up above.